api/lib/domain/usecases/get-certification-cpf-countries.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function getCertificationCpfCountries({ certificationCpfCountryRepository }) {
```

Comment:
pareil ici, je sais que c'est pas compris / su de tout le monde, mais y'a une forme de lexique associé à la récupération de données à Pix. Pour une raison d'homogénéité, j'encourage pour le moment à continuer d'utiliser la nomenclature dédiée. 
En général le mot `get` fait référence à une seule donnée, tandis que `find` évoque plutôt la collection